core/src/components/cat-alert/cat-alert.scss

```diff
@@ -17,11 +17,16 @@
 
 .alert {
   font: inherit;
-  color: cat-token-wrap(var(--text));
   background-color: cat-token-wrap(var(--bg), $alpha: 0.1);
-  box-shadow: inset 0 0 0 1px cat-token-wrap(var(--border), $alpha: 0.2);
-  border-radius: cat-border-radius('m');
-  padding: 0.75rem 1rem;
+  border-radius: cat-border-radius('l');
+  padding: 1.25rem;
+  display: flex;
+  flex-direction: row;
+  gap: 0.5rem;
+}
+
+::slotted(*) {
```

Comment:
Does this apply to all slotted content or only to direct children? I am always a bit cautious with * operator because of leaking styles everywhere

Reply:
Only the direct children but still, I changed the selector because I noticed that it would be more complicated to add children